guides/common/modules/con_deploying-virt-who-configurations.adoc

```diff
@@ -0,0 +1,10 @@
+[id="deploying-virt-who-configurations_{context}"]
+= Deploying virt-who configurations
+
+After you create a virt-who configuration, you download a script from the {ProjectWebUI} to deploy the configuration.
+
+The script installs virt-who and creates the local and global virt-who configuration files.
+
```

Comment:
Should we mention you can deploy the script on the `{ProjectWebUI}` with hammer as well? 
![screenshot-ip-10-0-167-73 rhos-01 prod psi rdu2 redhat com-2025 01 09-11_12_13](https://github.com/user-attachments/assets/48b67cac-bf52-452c-8606-416e2e836cb6)

Reply:
Sounds good.